src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -24,6 +24,7 @@
 public class ParserUtil {
 
     public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
+    public static final String EMPTY_ARGUMENT = "NA";
```

Comment:
I think it's best to put this constant in the `DateOfBirth` class since it's specific to DOB.

Perhaps also change its name to `NO_DOB` so that it is self-explanatory when used in other parts of the code.